docs/notes/2025.11.10.06.24.00.md

```
@@ -0,0 +1,19 @@
+to breathing even...
+
+the adhd mind, the modern mind.
+
+I have to get my brain on straight.
+it already is on straight.
+
+the opencode sessions should be locally inspectable
+the opencode state files should be a part of the local scope.
+`.opencode` should contain an isolated cache
+
+it has far too much global state.
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)